### DISCUSSION TITLE
Tile Card - Feature for "switch" entities

### DISCUSSION THREAD

**Discussion by ChristophCaina** [home-assistant/frontend#15346] [Other feature requests] [answered]
Hi,
at the moment I am redesigning my Dashboards and replace many elements with the new Tile Cards.
While I can use the tile card as a "button" (action = toggle) - I would like to see an option to use the tile card also for switch Entities.
Right - It could be done just like a button - and setting the action to toggle, but a nice feature addon would be to have the ability to show the toggle-slider like it is the case for entitie cards:

![grafik](https://user-images.githubusercontent.com/26391061/216812737-6d579722-bab7-4ff7-8f88-58d61aed200e.png)

**Comment by Mariusthvdb** [comment]
wow this is ancient, and still valid...

let me bump this, we really would be able to use a Tile card with a horizontal toggle in the features (and not have to open the more-info to reveal the huge vertical toggle)

idea like the https://www.home-assistant.io/dashboards/features/#humidifier-toggle 

<img width="720" alt="Scherm­afbeelding 2025-02-11 om 08 28 42" src="https://github.com/user-attachments/assets/41e4cd38-3dec-478b-9aaf-fb46b860b15d" />

and add that feature to toggled entities, like switch, input_boolean and light please?

**Reply by ChristophCaina** [reply]
just noticed, that this would also be a nice feature for the tile card, if you are using a "counter-helper".
Right now, you have to increase / decrease / reset the counter either in the "more info" dialog, via automation - or by calling the action for the counter, which means - if you want to do it within the UI:

Add the TileCard + configure a "primary action" - or place button cards next by. ...
So, having a feature like the below, would be great

```
features:
    - type: counter-action-modes
    action_modes:
      - increase
      - decrease
      - reset
```

**Reply by Mariusthvdb** [reply]
but that would be for another type of entity, so please write that up in a separate topic, not to bloat this one about Switch entities?
there is also a generic 'features for Tile card' discussion. But as you will see, it wont develop because too wide spread.

Separate topic for separate ideas works best

**Comment by jpbede** [comment, marked as answer]
https://github.com/home-assistant/frontend/pull/24325

**Reply by Mariusthvdb** [reply]
wonderful! thank you so very much

will this also work for/be added to booleans, and other on/off entities, like eg a light?

_____
as a sidebar:
would be cool if we would 'simply' have the toggle option, and not have to set it per device type 

so in stead of:
```
type: tile
entity: switch.example_switch
features:
  - type: switch-toggle
```

do
```
type: tile
entity: switch.example_switch
features:
  - type: toggle
```
or
```
type: tile
entity: light.example_light
features:
  - type: toggle
```
and
```
type: tile
entity: input_boolean.example_boolean
features:
  - type: toggle
```

(also thinking this with modding them in mind, via card-mod. We now have to target those all specifically per device-type, while a generic feature name would suffice and be much easier/cleaner)

`hui-fan-speed-card-feature`
`hui-humidifier-toggle-card-feature`
`hui-target-humidity-card-feature`
etc etc

would be awesome if in the future we could see just

`hui-target-card-feature`
`hui-toggle-card-feature`

etc etc....

If you're open to that thought, ld be glad to write up a separate FR/Discussion for that. It's related to this here, but a bit more generic too.. ;-)

**Reply by jpbede** [reply]
Yea, agreed! A comment was made on the PR as well. We rename the feature and make it usable for at least `input_boolean` as well.
